koswat/strategies/strategy_input.py

```diff
@@ -10,3 +11,17 @@ class StrategyInput:
     strategy_reinforcements: list[StrategyReinforcementInput]
     reinforcement_min_buffer: float
     reinforcement_min_length: float
+
+    @property
+    def reinforcement_min_cluster(self) -> int:
+        """
+        Returns the minimun length of a reinforcement type
+        along a traject, usually named as `cluster` throughout
+        the code.
+
+        Returns:
+            int: `Total length`
+        """
+        if math.isnan(self.reinforcement_min_buffer):
+            return -1
```

Comment:
Mostly to avoid throwing an exception whilst (indirectly) referring to this class. If you initialize the object and debug it, you are "raising" an exception already.
`-1` has always been a way of representing "invalid" integers.

Reply:
I see. But what if the caller doesn't check for -1?